web_src/less/_base.less

```
@@ -1732,6 +1732,20 @@ a.ui.card:hover,
   border-radius: .15em;
 }
 
+.attention() {
+  font-weight: unset;
+}
+
+.attention-note {
+  .attention;
+  color: var(--color-info-text);
+}
+
+.attention-warning {
+  .attention;
+  color: var(--color-warning-text);
```

```suggestion
.attention-note {
  font-weight: unset;
  color: var(--color-info-text);
}

.attention-warning {
  font-weight: unset;
  color: var(--color-warning-text);
```

Would prefer not to get fancy with Less.